File: AngularApp/InternalControl/src/app/Core/Models/questionModel.ts
import { BaseModel } from './baseModel';
import { GroupOfIndicators } from './groupOfIndicatorsModel';
import { Indicator } from './indicatorModel';
import { TypeOfForm } from './typeOfFormModel';
export class Question implements BaseModel{
    public Id: number;    
    public Name: string;
    public Answer: boolean;
    public GroupOfIndicators: GroupOfIndicators = new GroupOfIndicators();
    public Indicators: Indicator = new Indicator();
    public TypeOfForm: TypeOfForm = new TypeOfForm();
}

File: InternalControl.Core/Models/QuestionModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InternalControl.Core.Models
{
    public class QuestionModel : BaseModel
    {
        public virtual TypeOfForm TypeOfForm { get; set; }
     
        public virtual GroupOfIndicatorsModel GroupOfIndicators { get; set; }

        public virtual IndicatorsModel Indicators { get; set; }

        public virtual AnswersModel Answer { get; set; }

        public int TypeOfFormId { get; set; }

        public virtual ICollection<FormQuestion> FormQuestions { get; set; }
    }
}
